idf_component.yml

```diff
@@ -27,3 +27,15 @@ dependencies:
   idf: ">=5.1"
   mdns: "^1.1.0"
   chmorgan/esp-libhelix-mp3: "1.0.3"
+  esp-dsp: "^1.3.4"
+  espressif/esp_rainmaker: "^1.0.0"
+  espressif/rmaker_common: "^1.4.3"
+  espressif/esp_insights:  "^1.0.1"
+  espressif/qrcode: "^0.1.0~1"
+  espressif/esp32-camera: "^2.0.5"
```

Comment:
because camera driver is not used in any code in the core, I would ask you to remove it from here and leave it being handled by the lib-builder instead.

Reply:
Done